Looped/FlowControllers/PopularFlowController.swift

```diff
@@ -0,0 +1,26 @@
+import UIKit
+
+final class PopularFlowController: FlowController {
+
+    let rootControllersFactory = PopularFlowControllerFactory()
+    lazy var rootViewController: UINavigationController = {
+        navigationController.viewControllers = [rootControllersFactory.buildPopularViewController(delegate: self)]
+        
+        return navigationController
+    }()
+    
+    private lazy var navigationController: UINavigationController = {
+        return UINavigationController()
+    }()
+    
+    init() {
+        navigationController.show(rootViewController, sender: nil)
+    }
+}
+
+extension PopularFlowController: PopularViewControllerFlowDelegate {
+    func showDetails(gif: String) {
+        let detailsViewController = DetailsViewController(gif: gif)
```

Comment:
Tutaj tez powinno dzialac factory do tworzenia tego kontrollera

Reply:
Tzn dodać kolejną funkcję w PopularFlowControllerFactory, która tworzy DetailsViewController? Jeśli tak, to spoko - tylko będę to duplikował w trzech miejscach (jeśli miałby być dostępny DetailsViewController z każdego taba)